vault/resource_database_secret_backend_connection_test.go

```diff
@@ -1180,12 +1199,16 @@ resource "vault_database_secret_backend_connection" "test" {
 `, path, name, url, username, password, userTempl)
 }
 
-func testAccDatabaseSecretBackendConnectionConfig_redshift(name, path, connURL string) string {
-	return fmt.Sprintf(`
+func testAccDatabaseSecretBackendConnectionConfig_redshift(name, path, connURL string, isUpdate bool) string {
+	ret := fmt.Sprintf(`
```

Comment:
nit: maybe call it `config` or `result` ?

Reply:
Fixed in 57a0ec7